backward.hpp

```diff
@@ -97,6 +100,11 @@
 //  exception.
 //  - normally libgcc is already linked to your program by default.
 //
+// #define BACKWARD_HAS_LIBUNWIND 1
```

Comment:
Could be nice to complete the comment in the backward.cpp file line 18

Reply:
Done